source/dub/dependency.d

```diff
@@ -647,10 +657,10 @@ struct Version {
 		string m_version;
 	}
 
-	static @property Version minRelease() { return Version("0.0.0"); }
-	static @property Version maxRelease() { return Version(MAX_VERS); }
-	static @property Version masterBranch() { return Version(masterString); }
-	static @property Version unknown() { return Version(UNKNOWN_VERS); }
+	static @property Version minRelease() { return Version("0.0.0", true); }
+	static @property Version maxRelease() { return Version(MAX_VERS, true); }
+	static @property Version masterBranch() { return Version(masterString, true); }
+	static @property Version unknown() { return Version(UNKNOWN_VERS, true); }
```

Comment:
Is this that much of an improvement ? Maybe you'd like to provide those as `static immutable` instead, to avoid construction an instance of the struct over and over.

Reply:
Certainly not the top performance hog, but it was at the top of the profiler with the other things. Static immutable sounds like a good idea, not sure why that hasn't been the case.